mlflow/server/js/src/model-registry/components/ModelView.js

```diff
@@ -27,8 +27,8 @@ export class ModelView extends React.Component {
   static propTypes = {
     model: PropTypes.shape({
       name: PropTypes.string.isRequired,
-      creation_timestamp: PropTypes.number.isRequired,
-      last_updated_timestamp: PropTypes.number.isRequired,
+      creation_timestamp: PropTypes.string.isRequired,
+      last_updated_timestamp: PropTypes.string.isRequired,
```

Comment:
Fix prop types to `string` because `StrictJsonBigInt.parse` converts `creation_timestamp` and `last_updated_timestamp` to string (from int64).